mobile/src/main/java/ch/epfl/sdp/mobile/application/StoreDocuments.kt

```diff
@@ -64,6 +64,14 @@ data class ChessDocument(
     val moves: List<String>? = null,
     val whiteId: String? = null,
     val blackId: String? = null,
+    val lastUpdatedAt: Long? = null,
+    val metadata: ChessMetadata? = null
```

Comment:
Missing documentation for these two new fields.

Reply:
yes